Geometry/TrackerNumberingBuilder/test/trackerModuleNumberingDD4hep_cfg.py

```diff
@@ -0,0 +1,64 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@ianna , @fabiocos Is there any working alternative to this test currently available? Running it at `CMSSW_11_0_0_patch1` and the latest IB produces the following:

```
----- Begin Fatal Exception 24-Aug-2020 16:54:52 CDT-----------------------
An exception of category 'MustUseESGetToken' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'p1'
   [2] Calling method for module ModuleNumbering/'prod'
   [3] Using EventSetup component TrackerTopologyEP/'trackerTopology' to make data TrackerTopology/'' in record TrackerTopologyRcd
   [4] Using EventSetup component TrackerParametersESProducer/'DD4hep_trackerParameters' to make data PTrackerParameters/'' in record PTrackerParametersRcd
Exception Message:
Called EventSetupRecord::get without using a ESGetToken.
 While requesting data type:DDCompactView label:''
----- End Fatal Exception -------------------------------------------------
```

Reply:
`TrackerParametersESProducer` needs to be changed to use `ESGetToken`
https://github.com/cms-sw/cmssw/blob/86caca1fddb13f3506f3da1df5871c74a88f7149/Geometry/TrackerGeometryBuilder/plugins/dd4hep/TrackerParametersESProducer.cc#L52-L53
I suppose it is not used in any production workflow (otherwise should have caught it long ago)

Reply:
@makortel, not yet used in production workflows, but will be once the dd4hep migration is completed. Issue addressed here:
https://github.com/cms-sw/cmssw/pull/31226.

@vargasa the test `Geometry/TrackerNumberingBuilder/test/trackerModuleNumberingDD4hep_cfg.py ` is in any case unrunnable for me:
```
----- Begin Fatal Exception 25-Aug-2020 09:29:05 CEST-----------------------
An exception of category 'EventSetupConflict' occurred while
   [0] Calling beginJob
Exception Message:
two EventSetup Producers want to deliver type="GeometricDet" label=""
 from record IdealGeometryRecord. The two providers are 
1) type="TrackerGeometricDetESModule" label="DD4hep_trackerNumberingGeometry"
2) type="TrackerGeometricDetESModule" label="trackerNumberingGeometry"
Please either
   remove one of these Producers
   or find a way of configuring one of them so it does not deliver this data
   or use an es_prefer statement in the configuration to choose one.
----- End Fatal Exception -------------------------------------------------
```

after solving this there are still other problems related to the `IdealGeometryRecord` that I did not address. I suppose this configuration needs to be heavily cleaned up.

Reply:
We can add: 
```
process.preferDD4hep = cms.ESPrefer("TrackerGeometricDetESModule","trackerNumberingGeometry")
```
After loading the geometry sequences, but still:
```
----- Begin Fatal Exception 02-Sep-2020 13:28:53 CDT-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'p1'
   [2] Calling method for module ModuleNumbering/'prod'
   [3] Using EventSetup component TrackerGeometricDetESModule/'trackerNumberingGeometry' to make data GeometricDet/'' in record IdealGeometryRecord
   [4] Running EventSetup component TrackerGeometricDetESModule/'trackerNumberingGeometry
Exception Message:
No data of type "DDCompactView" with label "" in record "IdealGeometryRecord"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
```